libenv/unix_iface.c

```diff
@@ -657,9 +657,13 @@ static void FindV6InterfacesInfo(EvalContext *ctx)
 
                 if ((IsIPV6Address(ip->name)) && ((strcmp(ip->name, "::1") != 0)))
                 {
+                    char prefixed_ip[CF_MAX_IP_LEN] = {0};
```

Comment:
@vpodzime I think the proper one would be `CF_MAX_IP_LEN + sizeof("ipv6_")`

Reply:
It doesn't really matter because ``CF_MAX_IP_LEN`` is 64 for some unspecified reason, but yeah, this code shouldn't blindly rely on the current definition. Fixed.